### DISCUSSION TITLE
梅林进程启动失败

### DISCUSSION THREAD

**Discussion by moonan3019** [messense/aliyundrive-webdav#156] [Q&A]
梅林固件 开启插件失败
![Screenshot_20211117-214948](https://user-images.githubusercontent.com/70310435/142213788-e6b06ffc-968c-43bb-b01f-34468306962f.png)
![Screenshot_20211117-214728](https://user-images.githubusercontent.com/70310435/142213812-a0e12eff-2354-4dd8-9d22-81629c5fd7b2.png)
Log:
2021年11月17日 21:24:01: 当前已进入aliyundrivewebdav_config.sh
2021年11月17日 21:24:01: 当前已进入aliyundrivewebdavconfig.sh执行
2021年11月17日 21:24:01: 先结束进程
2021年11月17日 21:24:01: 参数为：9998 -r 0f0b6226c3694816bd3a19xxxxx4 --root / -S 10485760 --cache-size 1000 --auth-user huguyue --auth-password 159357
2021年11月17日 21:24:06: aliyundrive 进程启动失败！请检查参数是否存在问题，即将关闭
2021年11月17日 21:24:06: 失败原因：
BBABBBBC

Token我是网页获取的

**Comment by messense** [comment]
看看 `/tmp/upload/aliyundrivewebdav.log` 有没有错误日志。

**Reply by moonan3019** [reply]
我下面贴的log就是

**Reply by messense** [reply]
No, 那只是 web 界面的日志，需要程序的日志
https://github.com/messense/aliyundrive-webdav/blob/81b6b534d09b115a4d4c1cc454d2307997a30b1d/merlin/arm384/scripts/aliyundrivewebdavconfig.sh#L36

**Reply by moonan3019** [reply]
![Screenshot_20211117-220821.png](https://user-images.githubusercontent.com/70310435/142215833-1f5e2907-6208-4b6a-aee2-37b98a5b9830.png)

**Reply by moonan3019** [reply]
是这个那

**Reply by messense** [reply]
/tmp/upload/aliyundrivewebdav.log

**Reply by messense** [reply]
你图里面的是 /tmp/upload/aliyundrivewebdav**config**.log

**Reply by moonan3019** [reply]
不好意思 搞错了 ![Screenshot_20211117-221131.png](https://user-images.githubusercontent.com/70310435/142216380-05ea93ac-c54b-49e9-b504-dffe27888e73.png)这个log文件里面啥也没有啊

**Reply by messense** [reply]
直接 `/koolshare/bin/aliyundrive-webdav -p 9998 -r 0f0b6226c3694816bd3a19xxxxx4 --root / -S 10485760 --cache-size 1000 --auth-user huguyue --auth-password 159357` 命令行执行看下输出吧

**Reply by moonan3019** [reply]
我试一下

**Reply by moonan3019** [reply]
这个要在哪里执行啊 要开一个ssh吗

**Reply by messense** [reply]
是的

**Reply by moonan3019** [reply]
需要在前面加什么吗 还是直接复制进去回车啊 命令行我太陌生了@

**Reply by messense** [reply]
复制进去，改下 -r 后面的值为真实的 refresh_token 后回车。

**Reply by moonan3019** [reply]
-sh: /koolshare/bin/aliyundrive-webdav: not found

**Reply by messense** [reply]
hmm，你安装的哪个版本文件？

**Reply by moonan3019** [reply]
这个我是从路由器软件中心下载的

**Comment by moonan3019** [comment]
![mmexport1637159511400.png](https://user-images.githubusercontent.com/70310435/142219933-7aa839b9-3707-499e-adff-22fcb52f5c50.png)

**Reply by messense** [reply]
那这个版本可能是软件中心他们打包的也许有些文件位置的改动，命令行执行 `ls -lrth /koolshare/bin` 看看？

**Reply by moonan3019** [reply]
hugue@RT-AC68P-264C:/tmp/home/root# ls -lrth /koolshare/bin                                                                                                                                                                                                                                                                                                              
ls: /koolshare/bin: No such file or directory

**Reply by messense** [reply]
看起来文件结构是不一样的，建议咨询下固件作者。

**Reply by moonan3019** [reply]
就是说固件的文件结构不同导致配置不上了是吧 我看我在插件里看插件版本都是空的 我去git上下载的版本都是提示安装目录不对装不上

**Reply by messense** [reply]
可能是的，没用过这个版本的梅林固件，目前适配的都是 koolshare 版本的。

**Reply by moonan3019** [reply]
那我明天尝试把插件文件里的安装位置改一下重新打包试试 非常感谢

**Reply by moonan3019** [reply]
http://router.asus.com:26789/files/jffs/softcenter/
这个路径下我都需要改哪些文件呢

**Reply by moonan3019** [reply]
http://router.asus.com:26789/files/jffs/softcenter/scripts/
这个路径 刚才那个少了一层

**Reply by messense** [reply]
不清楚，你给的地址只有你的网络能访问，暂时没法帮你。

**Reply by moonan3019** [reply]
等我把这个固件的路径搞明白再来请教 非常感谢
